src/forms/Feedback.jsx

```diff
@@ -0,0 +1,106 @@
+                <Form.Group as={Col} lg="6">
+                    <Form.Label column="lg">Категория:</Form.Label>
+                    <Form.Select
+                        size="lg"
+                        type="category"
+                        name="category"
+                        value={formik.values.category}
+                        onChange={formik.handleChange}
+                        onBlur={formik.handleBlur}
+                        isValid={formik.touched.category && !formik.errors.category}
+                        isInvalid={formik.touched.category && !!formik.errors.category}
+                    >
+                        <option id="0" value="" disabled hidden>
+                            Выберите категорию
+                        </option>
+                        {categories.map((category) => {
+                            return (
+                                <option value={category} id={category}>
+                                    {category}
+                                </option>
+                            )
+                        })}
+                    </Form.Select>
+                </Form.Group>
+            </Row>
+            <Row className="mb-3">
+                <Form.Group as={Col} lg="8">
+                    <Form.Label column="lg">Суть обращения:</Form.Label>
+                    <Form.Control
+                        size="lg"
+                        as="textarea"
+                        name="text"
+                        placeholder="Вы можете описать проблему или задать вопрос тут"
+                        value={formik.values.text}
+                        onChange={formik.handleChange}
+                        onBlur={formik.handleBlur}
+                        isInvalid={formik.touched.text && !!formik.errors.text}
+                        isValid={formik.touched.text && !formik.errors.text}
+                        rows={8}
+                    />
+                </Form.Group>
+            </Row>
+            <Row className="mb-3">
+                <Form.Group as={Col} lg="8">
+                    <Form.Label column="lg">Электронная почта:</Form.Label>
+                    <Form.Control
+                        size="lg"
+                        type="email"
+                        name="email"
+                        placeholder="Введите адрес почты"
+                        value={formik.values.email}
+                        onChange={formik.handleChange}
+                        onBlur={formik.handleBlur}
+                        isInvalid={formik.touched.email && !!formik.errors.email}
+                        isValid={formik.touched.email && !formik.errors.email}
+                    />
+                    <Form.Text muted>
+                        Оставьте вашу электронную почту, если вы хотите получить ответ на обращение.
+                    </Form.Text>
+                </Form.Group>
+            </Row>
+            <Form.Group as={Row} className="my-3">
```

Comment:
А зачем здесь отступ вниз? Карточка уже и так предоставляет дефолтный паддинг
Давай уберем:
```suggestion
            <Form.Group as={Row}>
```
(верхний отступ не работает из-за того что у элемента сверху mb-3 )